odm2api/ODM2/services/readService.py

```diff
@@ -615,7 +615,7 @@ def getAffiliations(self, ids=None, personfirst=None, personlast=None, orgcode=N
 
     # Results
     def getResults(self, ids=None, type=None, uuids=None, actionid=None, simulationid=None, sfid=None,
```

Comment:
While we are at this, can `type` change to `restype`?

Reply:
Also, is `sfid` being used anymore? If not, can it be removed?

Reply:
@lsetiawan I don't want to just delete the parameters because I know there are people out there who are using these functions, and just removing them would cause their code to stop working. So what I am doing is adding a warning message to let them know the function call will be changing, so that people have a little time to make the change. then the next release we do, we can delete them. but yes I can add `restype`.  and a warning message to `type`,

Reply:
Ah okay. Thanks for the explanation. That makes sense 😄